src/python/pants/backend/nfpm/native_libs/elfdeps/analyze.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2025 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+"""A standalone script that uses elfdeps to analyze ELF binaries/libraries.
+
+Rule code must treat this script as a `resource`, running it as a subprocess in a sandboxed venv.
+Rule code must not import anythin from this script or use it as a `python_source`.
```

Comment:
```suggestion
Rule code must not import anything from this script or use it as a `python_source`.
```

But this isn't worth a CI round to fix, so feel free to fix in a future change, even an unrelated one.

Reply:
Oops. I'll put that in the next PR. Thanks!